integration_tests/seeds/hubspot_engagement.csv

```diff
@@ -0,0 +1,11 @@
+id,_fivetran_synced,active,activity_type,created_at,last_updated,owner_id,portal_id,timestamp,type
+19732910159,2023-11-02 13:50:33.715000 UTC,,,,,,4703379,,CALL
```

Comment:
I think we will need to remove these UTC strings for Snowflake Buildkite errors to be fixed.